net.solarnetwork.node.hw.energymeter.mock/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,24 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: EnergyMeter Mock
+Bundle-SymbolicName: net.solarnetwork.node.datum.energymeter.mock
+Bundle-Version: 1.0.0
+Bundle-Vendor: SolarNetwork
+Bundle-RequiredExecutionEnvironment: JavaSE-1.6
+Import-Package: net.solarnetwork.node,
+ net.solarnetwork.node.dao;version="1.8.0",
+ net.solarnetwork.node.domain;version="1.11.0",
+ net.solarnetwork.node.job;version="1.13.4",
+ net.solarnetwork.node.settings;version="1.10.0",
+ net.solarnetwork.node.settings.support;version="1.8.0",
+ net.solarnetwork.node.support;version="1.14.0",
+ net.solarnetwork.node.util;version="1.7.2",
+ net.solarnetwork.util;version="1.28.0",
+ org.quartz;version="2.2.3",
```

Comment:
Package import minimum versions are OK for SolarNetwork internal packages, but I prefer version ranges for all 3rd party packages ([example](https://github.com/SolarNetwork/solarnetwork-node/blob/d7e75dbb1236d4b4c3984e1eeeff0547dd7f11ad/net.solarnetwork.node.datum.deson.sdm/META-INF/MANIFEST.MF#L21-L30)).